lib/b2b/index.ts

```diff
@@ -1,128 +1,87 @@
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+// MANUAL(exports)(TYPES)
+export type { SearchQueryOperand } from "./organizations";
+// ENDMANUAL(exports)
+
+export type { MemberOptions, MfaRequired } from "./mfa";
+
 export type {
-  B2BAuthenticationFactor,
-  MemberSession,
+  ActiveSSOConnection,
   Member,
-  SSORegistration,
   OAuthRegistration,
-} from "./shared_b2b";
+  Organization,
+  B2BOrganizationsResultsMetadata,
+  SSORegistration,
+  SearchQuery,
+  B2BOrganizationsCreateRequest,
+  B2BOrganizationsCreateResponse,
+  B2BOrganizationsDeleteRequest,
+  B2BOrganizationsDeleteResponse,
+  B2BOrganizationsGetRequest,
+  B2BOrganizationsGetResponse,
+  B2BOrganizationsSearchRequest,
+  B2BOrganizationsSearchResponse,
+  B2BOrganizationsUpdateRequest,
+  B2BOrganizationsUpdateResponse,
+} from "./organizations";
 
 export type {
-  B2BDiscoveryIntermediateSessionExchangeRequest,
```

Comment:
Looks like we lost lots of exports from this file. We need to reexport every type we want to be public. JS's export syntax is one thing I abhor, and one thing I think golang got very very right